packages/new-deploy/src/core/CoreContracts.ts

```diff
@@ -94,7 +97,7 @@ export default class EvmCoreDeploy extends AbstractCoreDeploy<config.EvmCoreCont
   }
 
   get replicas(): ReadonlyArray<string> {
-    return Object.keys(this.replicas);
+    return Object.keys(this.data.replicas!);
```

Comment:
avoid using `!` operator. instead use a `if (!whatever) throw new Error("...")`

Reply:
previously we've been using these, but they're bad hygiene and undermine the type system, so we should probably stop 😅

Reply:
makes total sense. fixed in [92dde9a](https://github.com/nomad-xyz/monorepo/pull/81/commits/92dde9a14ab7121d4d2abadcc316003c217d4c57)